projects/is-selectpicker/src/lib/is-selectpicker.component.ts

```diff
@@ -377,7 +378,7 @@ export class IsSelectpickerComponent implements ControlValueAccessor, OnInit, On
       this.filteredOptions = [];
     } else {
       this.filteredOptions = this._options.filter(item => {
-        return item.Value.toUpperCase().indexOf(newValue.toUpperCase()) !== -1;
+        return createFilterRegexp(newValue).test(item.Value);
```

Comment:
Did you test this code? `is-select` implements it a bit differently, because it generates `FilterValue` property for each item. What if filter is 'tomas' and item.Value is 'Tomáš'?

Reply:
I've tried to adapt implementation from `is-select` without making any major changes to `is-selectpicker`. Still, it takes into account both case insensitive filtering and stripping down diacritics, as shown on a screenshot below. Or am I missing something? 

![Snímek z 2019-06-28 08-30-10](https://user-images.githubusercontent.com/15877754/60322207-26a8be00-997f-11e9-8de9-024b441dd149.png)

Reply:
Sorry, now I understand. No, I forgot about that type of situation, I will fix it

Reply:
It is now working as desired

![Snímek z 2019-06-28 09-18-19](https://user-images.githubusercontent.com/15877754/60324741-e0a32880-9985-11e9-852d-c43bf8d66bc2.png)